src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from '../components/PersonLink/PersonLink';
+import { Loader } from '../components/Loader/Loader';
+
+function findPersonByName(name: string | null, people: Person[]) {
+  return people.find(person => person.name === name);
+}
+
+export const PeoplePage = () => {
+  const { personSlug } = useParams();
+  const [people, setPeople] = useState<Person[]>();
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setErrorMessage('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+  return (
+    <>
+      <h1 className="title">People Page</h1>
```

Comment:
add upper `<div className="block">`

Reply:
Why?
JSX fragment doing the same job here